css/index.css

```diff
@@ -218,50 +234,49 @@ header nav a:hover {
     width: 100%;
     flex-direction: column-reverse;
   }
-
   header nav {
-    width: 400px;
+    width: 100%;
+    margin: 0%;
     display: flex;
     flex-direction: column;
   }
-
   header nav a {
-    width: 400px;
+    width: 100%;
     text-align: center;
     border: solid #808080 1px;
   }
-
   header .logo {
     align-self: center;
-    margin: 25px 0px 0px 20px;
+    margin: 6.2% 0% 0% 5%;
```

Comment:
I mentioned this in the solution lecture for Thursday night, but the general rule of thumb is that the top and bottom margins are to be in px while the sides are in percentages. Top/bottom are more of aesthetic properties rather than for them to be fluid/responsive. Upon seeing that, making everything into percentages can be quite tedious so kudos to that! You have a lot of great implementation from new material that haven't been covered to just having a great looking site.